features/vendor_views_closed_auction.feature

```diff
@@ -38,6 +38,11 @@ Feature: Vendor views closed auctions
     And I should see when the auction ended
     And there should be meta tags for the closed auction
 
+  Scenario: Winning vendor was paid
```

Comment:
This seems to be a test for a guest who isn't logged in. Maybe we should consider a `background` for all of these scenarios to guarantee the user is logged in?